docs/Makefile

```diff
@@ -2,11 +2,13 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W -n
-SPHINXBUILD   = sphinx-build
-PAPER         =
-BUILDDIR      = _build
-PULP_URL      = "http://localhost:24817"
+SPHINXOPTS    		  = -W -n
+SPHINXBUILD   		  = sphinx-build
+PAPER         		  =
+BUILDDIR      		  = _build
+DIAGRAM_BUILD_DIR     = _diagrams
+DIAGRAM_SOURCE_DIR    = diagrams_src
```

Comment:
Is this one of those instances when the formatting on github lies?  Otherwise I'd prefer everything lined up.

Reply:
This file is [generated by the plugin template](https://github.com/pulp/plugin_template/blob/master/templates/docs/docs/Makefile.j2). I can go fix it in the plugin_template. My personal preference is to not line anything up. wdyt?

Reply:
Here we go https://github.com/pulp/plugin_template/pull/288

Reply:
I also prefer not lining up, because if you add one item you only need to touch that one line and not realign the whole block.
This is again about small commits that show you precisely what changed.